infrasim/global_status.py

```diff
@@ -0,0 +1,48 @@
+import os

```

Comment:
Hi, @WinnieXU1 ,

At first glance, I think global status is a behavior and doesn't need to be a module. 
To think further, you can actually make a module which is responsible for global monitoring:
- capture process status;
- capture port utilization;
- ...
I'm looking forward to a better design (could be a incomplete implementation) on this module.

About to display the information, vagrant shows a more concise style which worth learning.